doc/source/API/SetupTemplates.rst

```diff
@@ -0,0 +1,176 @@
+Setup Templates
+===============
+This page contains all the Setup Default Templates values and keys. it can be used to edit a setup after it is created
+Example
```

Comment:
```suggestion
This page contains all the Setup Default Templates values and keys. it can be used to edit a setup after it is created. 
 Here's an example:
```